### PR TITLE
NO-JIRA update `artemis create` samples in README.html and using-server.md

### PR DESCRIPTION
**Pull request by jdanekrh**

### CODE REVIEW

File: docs/user-manual/en/using-server.md [hunk 1]
```diff
@@ -200,7 +213,11 @@ For a full list of updated properties always use:
              Disable the STOMP specific acceptor.
 
          --no-web
-             This will remove the web server definition from bootstrap.xml
+             remove the web-server definition from bootstrap.xml
```
Comment (jdanekrh) [inline docs/user-manual/en/using-server.md]: maybe capitalize the r?

Comment (mtaylor) [inline docs/user-manual/en/using-server.md]: Yes Capitalise the R

Comment (mtaylor) [inline docs/user-manual/en/using-server.md]: Clearly demonstrated by my misuse of capital letters. ;)

Comment (jdanekrh) [inline docs/user-manual/en/using-server.md]: done

**Pull request closed without merging**